pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -36,11 +38,15 @@ import (
 
 var logger = capnslog.NewPackageLogger("github.com/rook/rook", "op-mgr")
 
+var prometheusRuleName = "prometheus-ceph-VERSION-rules"
+var prometheusRuleFile = "/etc/ceph-monitoring/prometheus-ceph-VERSION-rules.yaml"
```

Comment:
A couple small suggestions:
- To avoid the minor duplication of `prometheus-ceph-VERSION-rules`, could you format the prometheusRuleName into the prometheusRuleFile? 
- It would be nice if the filename included a `v` so it's obvious `14` is the version. For example `prometheus-ceph-v14-rules.yaml`

Reply:
done.